sysprep/instance_setup.ps1

```diff
@@ -245,6 +245,24 @@ function Create-GCEStartup {
   $folder.RegisterTaskDefinition('GCEStartup',$task,6,'System',$null,5) | Out-Null
 }
 
+function Optimize-PowershellAssemblies {
+  # NGEN powershell assembly, improves startup time of powershell by 10x
+  $old_path = $env:path
```

Comment:
This is pretty aggressive, do you have to overwrite the path? This only really matters if you run this function in the background.

Reply:
No, that was just simplest. I've adjusted.